wordle-web/components/hamburgerButton

```diff
@@ -0,0 +1,75 @@
+<v-btn icon @click="toggleDialog">
```

Comment:
This file should have the extension of `.vue` and should be written in the typical vue component format if you intend to use it. Otherwise, you should be sure to remove this file from the codebase.

For example:
```vue
<template>
  <!-- Vue UI Code Here -->
</template>

<script lang="ts">
import { Component, Vue,} from 'vue-property-decorator'

@Component({ components: {} })
export default class ClassNameHere extends Vue {
 // Typescript code here
}
</script>
```